CONTRIBUTING.md

```diff
@@ -1,16 +1,518 @@
-# Contributing
+# Contributing to the Conjur Google Cloud Marketplace Application
 
-For general contribution and community guidelines, please see the [community repo](https://github.com/cyberark/community).
+Thank you for your interest in the Conjur Google Cloud Marketplace Application. Before contributing, please take a moment to read and sign our <a href="https://github.com/cyberark/community/blob/master/documents/CyberArk_Open_Source_Contributor_Agreement.pdf" download="secretless-broker_contributor_agreement">Contributor Agreement</a>.
+This provides patent protection for all Secretless Broker users and allows CyberArk to enforce its license terms. Please email a signed copy to
+<a href="oss@cyberark.com">oss@cyberark.com</a>.
 
-## Contributing
+For general contribution and community guidelines, please see the [CyberArk community repo](https://github.com/cyberark/community).
 
-1. [Fork the project](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
-2. [Clone your fork](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
-3. Make local changes to your fork by editing files
-3. [Commit your changes](https://help.github.com/en/github/managing-files-in-a-repository/adding-a-file-to-a-repository-using-the-command-line)
-4. [Push your local changes to the remote server](https://help.github.com/en/github/using-git/pushing-commits-to-a-remote-repository)
-5. [Create new Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork)
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Pull Request Workflow](#pull-request-workflow)
+- [Building, Deploying, and Testing](#building-deploying-and-testing)
+- [Releasing](#releasing)
+- [Publishing to Google Marketplace](#publishing-to-google-marketplace)
+
+## Prerequisites
+
+### Prerequisites for Building and Testing
+
+In order to contribute to the Conjur Google Marketplace Application, you
+must be able to build and test the required component images. To do this,
+you will need:
+
+* A development host with Docker server running.
+* A Kubernetes platform on which to deploy the application and test containers. For building and testing, this does not need to be a Google Kubernetes Engine (GKE) cluster.
+* A Docker registry to which to push images. This can be either:
+  - A Dockerhub registry, or...
+  - Access to the conjur-cloud-launcher-onboard GCP project, which
+    provides access to the gcr.io/conjur-cloud-launcher-onboard
+    Google Cloud Registry (GCR). To test if you have access to this GCP
+    project, try browsing to this location:
+    https://console.cloud.google.com/home/dashboard?project=conjur-cloud-launcher-onboard
+
+### Prerequisites for Publishing to Google Marketplace 
+
+In order to publish images to the Goggle Marketplace, you will need both:
+
+* Access to the `conjur-cloud-launcher-onboard` Google Cloud Platform (GCP) project
```

Comment:
Not a blocker but you should use `-` for toplevel items in a list since we do that everywhere and `*` can easily become a style formatting start/stop if indentation is messed up.

Reply:
Fixed.